README.md

```diff
@@ -6,9 +6,11 @@ src="https://cdn.rawgit.com/mlpack/mlpack.org/e7d36ed8/mlpack-black.svg" style="
 
 <h5 align="center">
   <a href="http://mlpack.org">Home</a> |
-  <a href="http://www.mlpack.org/docs/mlpack-git/doxygen/index.html">Documentation</a> |
-  <a href="http://www.mlpack.org/involved.html">Community</a> |
-  <a href="http://www.mlpack.org/help.html">Help</a> |
+  <a href="http://www.mlpack.org/docs.html">Documentation</a> |
+  <a href="http://www.mlpack.org/doc/mlpack-git/doxygen/index.html">Doxygen</a>
+|
```

Comment:
Not sure this character has to be on a separate line.

Reply:
Ah, yeah, it accidentally wrapped.  Fixed now.